src/schema/v2/me/auction_results_by_followed_artists.ts

```diff
@@ -103,6 +102,16 @@ const AuctionResultsByFollowedArtists: GraphQLFieldConfig<
       return auctionLotsLoader(diffusionArgs).then(
         ({ total_count, _embedded }) => {
           const totalPages = Math.ceil(total_count / size)
+
+          // enrich result with artist data
+          const items = _embedded.items.map((auctionResult) => {
+            const artist = followedArtists.find(
+              (artist) => artist.artist._id == auctionResult.artist_id
+            )
+            auctionResult.artist = artist?.artist
+            return auctionResult
+          })
```

Comment:
Note: this doesn't fetch any additional data and just maps the artists from Gravity to the auction results by `artist id`. We could check if the artist field is requested but I guess mapping the data shouldn't slow the query down very much.